drizzlepac/processInput.py

```diff
@@ -557,6 +557,11 @@ def process_input(input, output=None, ivmlist=None, updatewcs=True,
         if '_flc.fits' in img:
             drz_extn = '_drc.fits'
             break
```

Comment:
Do you really want a "break" here on Line 559 and then again in Line 564?  It looks like there is a whole list of image names to process.

Reply:
The assumption, as you can tell, is that all the inputs will have the same suffix (flc.fits vs flt.fits) on input.  This code gets used (almost?) exclusively for either a single observation or a single association, so that assumption will hold true.  As for WFPC2 data, there is one type which makes this amount of logic overkill (come to think of it). the output from this function gets used to define the single output filename to be generated by AstroDrizzle, so we only need a single value.  Given that, it may just be easier to check the suffix of the first input member and be done with it rather than looking for an exception further down the list of inputs.